### DISCUSSION TITLE
Does not work in Opera browser

### DISCUSSION THREAD

**Discussion by bsdesigner** [toutjavascript/Fooocus-Log-Viewer#19] [General]
Today's generations he can't see:
![Снимок экрана 2024-04-16 143942](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/70bdbd76-d03b-4ab4-ae9b-1317a9ba7064)

He sees images, somehow selectively. In some old folders it sees them, and in others it does not:
![Снимок экрана 2024-04-16 144445](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/cab11e26-1fd4-48eb-8f38-e6140493d25a)

**Comment by toutjavascript** [comment]
Hello.
It works on my device.
Can you check in an other browser ?

**Reply by bsdesigner** [reply]
Before a certain date, it sees the contents of folders, but after 20.03.2024, it no longer sees anything. I tried it today in Opera, Chrome and Edge browsers.

**Reply by toutjavascript** [reply]
It seems strange. I can see all folder on my device.
Can you send me the log.html file of a folder that doesn't work.

**Reply by bsdesigner** [reply]
[log.zip](https://github.com/toutjavascript/Fooocus-Log-Viewer/files/15023370/log.zip)

**Reply by toutjavascript** [reply]
Ok, I see what happens.
You choose jpeg generation image format. 
I will fix it soon !

**Reply by bsdesigner** [reply]
Ah, that's it, now I see.... I actually switched to .jpeg to save space, right around that time ))))

**Comment by bsdesigner** [comment]
And then there's the problem... I can't integrate Log-Viewer into Fooocus:
![Снимок экрана 2024-04-18 141437](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/96d19e14-7119-4390-810e-678941f862f9)

I changed the ui_gradio_extensions.py file, but nothing changed:
![Снимок экрана 2024-04-18 142046](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/fc4e9e18-2a99-42f2-b13c-7d54b9f059a0)

Maybe it's because I'm using Fooocus in Stability Matrix? Or am I just doing something wrong?

**Reply by toutjavascript** [reply]
You should add the viewer.js include on line 43
```
    head += f'<meta name="samples-path" content="{samples_path}"></meta>\n'
    head += f'<script type="text/javascript" src="file=outputs/viewer.js"></script>\n'

    if args_manager.args.theme:
        head += f'<script type="text/javascript">set_theme(\"{args_manager.args.theme}\");</script>\n'

```

I will update the .js to optimize the include display

**Reply by bsdesigner** [reply]
Changed it, but nothing has changed:
![Снимок экрана 2024-04-18 160128](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/5c4ea2b9-e279-4134-a1af-66786b37deeb)

I'll wait for the viewer.js file to be updated...

**Reply by toutjavascript** [reply]
You can force clearing cache with CTRL + F5 (on windows) 
Links should appear

**Reply by bsdesigner** [reply]
The buttons appeared:

![Снимок экрана 2024-04-18 162933](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/00c42a1c-03c0-4c74-9a24-5d4824dba037)

But the images do not appear:

![Снимок экрана 2024-04-18 162544](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/8e215dcb-9996-4054-b014-54ea42698bdd)

![Снимок экрана 2024-04-18 162700](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/e09f0ee5-507a-4266-9512-79fd9ae61417)

**Reply by toutjavascript** [reply]
Argh, I don't understand
Do you really have the 1.5.5 release ?

**Reply by bsdesigner** [reply]
No, that was release 1.5.4. Release 1.5.5, I've only just now discovered....

**Reply by bsdesigner** [reply]
Now, with the new release, everything works fine:
![Снимок экрана 2024-04-19 145953](https://github.com/toutjavascript/Fooocus-Log-Viewer/assets/36730003/cef33bff-9152-424a-a15b-6a1ddf4b74fd)

Thank you so much for your work!
